x86_64-bootloader.json

```diff
@@ -16,7 +16,6 @@
     "os": "none",
     "features": "-mmx,-sse,+soft-float",
     "disable-redzone": true,
-    "panic": "abort",
-    "executables": true,
-    "relocation_model": "static"
```

Comment:
This may work at the moment, but I think it could break with any new rustc version without `"relocation_model": "static"`. With the default `dynamic` relocation model there has to be some code that initializes the GOT and performs other relocations. That it works right now may be because of the linker just so happening to be able to see that none of the called functions are exported and as such can't be interposed at runtime. This allows the use of pc-relative addressing instead of a PLT trampoline that accesses the GOT. There are no guarantees about which rust functions are exported or not.

Reply:
Thanks for the explanation! Would there be a way to keep the `relocation_model` field while suppressing the warnings or should I revert the last commit?

Reply:
I think it should be `relocation-model` with a `-` instead of `_`.

Reply:
The main branch had the same problem: https://github.com/rust-osdev/bootloader/pull/186

Reply:
Ah yes indeed! https://doc.rust-lang.org/rustc/codegen-options/index.html#relocation-model